bundles/org.openhab.binding.tr064/README.md

```diff
@@ -119,6 +119,11 @@ The call-types are the same as provided by the FritzBox, i.e. `1` (inbound), `2`
 | `wifi5GHzEnable`           | `Switch`                  |          | Enable/Disable the 5.0 GHz WiFi device.                        |
 | `wifiGuestEnable`          | `Switch`                  |          | Enable/Disable the guest WiFi.                                 |
 | `macOnline`                | `Switch`                  |     x    | Online status of the device with the given MAC                 |
+| `macIP`                    | `String`                  |     x    | IP of the device with the given MAC                            |
+| `macSignalStrength1` | `Number:Dimensionless`    |     x    | Wifi Signal Strength of the device with the given MAC. This is set in case the Device is connected to 2.4Ghz  |
```

Comment:
As the system channel type is of type `Number`, this needs to be `Number`, too.

Reply:
Yes you are right, I missed to update the readme documentation. The type for the channels is already of type Number.